### PR TITLE
feature/#165/sc-page-change

### PR DESCRIPTION
**Pull request by jherkyesse**
### What does this pull request aim to achieve?

- Need to update all styles and text based on the latest designs on Figma.
- Designs need to be more pixel perfect.

https://github.com/FightPandemics/FightPandemics/issues/165

### CODE REVIEW

**Comment by snowiesuet** [comment]
@robinv85 any updates with the eslint? It seems that most PR requests have eslint differences.

File: client/src/components/StepWizard/WizardNav.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import styled from "styled-components";
 import { Link } from "react-router-dom";
+import { mq } from "../../constants/theme";
```
Comment (anniegiang) [inline client/src/components/StepWizard/WizardNav.js:4]: Please use absolute imports. Should be `constants/theme`

**Comment by anniegiang** [comment]
@jherkyesse In the future please follow ALL the [instructions](https://github.com/FightPandemics/FightPandemics/blob/master/.github/pull_request_template.md) before submitting a PR.

File: client/src/templates/layouts/MobileTabs.js [hunk 2]
```diff
@@ -1,78 +1,91 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect } from "react";
```
Comment (anniegiang) [inline client/src/templates/layouts/MobileTabs.js:1]: I think we already have this component (#293), created by @Chimaobi-Johnson.

File: client/src/components/StepWizard/WizardNav.js (hunk 1 above)
Comment (jherkyesse) [reply client/src/components/StepWizard/WizardNav.js:4]: Ok I will adjust it.

**Pull request closed without merging**